cmd/govim/signs.go

```diff
@@ -9,26 +9,45 @@ import (
 // Using a sign group creates a separate namespace for all signs placed by govim
 const signGroup = "govim"
 
-// Name of different sign types used, only one at the moment, errors.
-const (
-	errorSign = "govimerr"
-)
+// signSuffix is used when naming the different sign types. It is appended to each
+// highlight type and passed in as name to sign_define().
+const signSuffix = "Sign"
+
+// Each sign in vim has a prio. If there are multiple signs on the same line, it
```

Comment:
I'm assuming "prio" is "priority"? If so, perhaps we can rename this - it's not a shortening I'm particularly familiar with and it doesn't seem worth the potential confusion for the sake of 4 chars 😄 (admittedly everywhere)

Reply:
fixed